gnosis/gp-swap-ui
# Add IPFS deployment

Adds an IPFS Github workflow, wich does a build and deploy to IPFS.

For now, no automatic release comments or pinning or any fancy stuff. We can re-iterate on this basic setup. 


![image](https://user-images.githubusercontent.com/2352112/115590814-ad0b5e00-a2d1-11eb-92b3-06360621145d.png)


## Test
1. Build: `yarn ipfs:build`
2. Publish to infura: `yarn ipfs:publish`

You should get a testable version like: https://ipfs.infura.io/ipfs/QmQZyQtdygS9RpjBY8BmytoaqH1JCACYuy3uorA3r38Bnf/

3. Make sure the IPFS is working as expected
